app/react/socket.js

```diff
@@ -1,17 +1,17 @@
 import io from 'socket.io-client';
 import { isClient } from 'app/utils';
 
-let socket = { on: () => {} };
+let _socket = { on: (_event, _listener) => {}, off: (_event, _listener) => {} };
+
 if (isClient) {
   //only websockets used, this allows for non sticky sessions on load balancer
-  socket = io({ transports: ['websocket'], upgrade: false });
+  _socket = io({ transports: ['websocket'], upgrade: false });
 }
 
-export default socket;
-
 const reconnectSocket = () => {
-  socket.disconnect();
-  socket.connect();
+  _socket.disconnect();
+  _socket.connect();
 };
 
+export const socket = _socket;
 export { reconnectSocket };
```

Comment:
This could have been `export { socket, recconetSocket }` and we avoid the underscored name, right?